### PR TITLE
Add support for Edge webdriver

### PR DESCRIPTION
**Pull request by Etienne42**
Add support for [Edge webdriver](https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/).

The following tests are failing:
- integration tests - page test for Edge - should support links and navigation
- integration tests - page test for Edge - should support switching windows
- integration tests - page test for Edge - should support popup boxes
- integration tests - page test for Edge - should support various mouse events
- integration tests - selection test for Edge - should support retrieving element properties
- integration tests - selection test for Edge - should support element actions

However, when using the webdriver on my own, I noticed that `Click` is not working most of the time, requiring a `DoubleClick` instead. Don't know yet if normal or an issue on the edge webdriver

### CODE REVIEW

File: agouti.go [hunk 1]
```diff
@@ -43,6 +43,22 @@ func ChromeDriver(options ...Option) *WebDriver {
...
 }
 
+// EdgeDriver returns an instance of a EdgeDriver WebDriver.
+//
+// Provided Options will apply as default arguments for new pages.
+// New pages will accept invalid SSL certificates by default. This
+// may be disabled using the RejectInvalidSSL Option.
+func EdgeDriver(options ...Option) *WebDriver {
+	var binaryName string
+	if runtime.GOOS == "windows" {
```
Comment (sclevine) [inline agouti.go]: Let's fail more gracefully on non-Windows OSs here. The method could return `nil`, for instance.

File: internal/integration/integration_suite_test.go [hunk 2]
```diff
@@ -14,6 +14,7 @@ var (
 	chromeDriver     = agouti.ChromeDriver()
 	seleniumDriver   = agouti.Selenium(agouti.Browser("firefox"))
 	selendroidDriver = agouti.Selendroid("selendroid-standalone-0.15.0-with-dependencies.jar")
+	edgeDriver       = agouti.EdgeDriver()
```
Comment (sclevine) [inline internal/integration/integration_suite_test.go]: This needs to be gated for non-Windows OSs.

File: agouti.go (hunk 1 above)
Comment (Etienne42) [inline agouti.go]: True, I forgot to handle the non-windows cases

**Comment by Etienne42** [comment]
@sclevine the tests are passing on go 1.7 but not on older version (an error changed).
The code also now handle the non-windows platform

**Comment by sclevine** [comment]
Could you update the failing test to use `MatchError` with a regex so it passes on all versions of Go?

**Comment by Etienne42** [comment]
@sclevine tests are now passing

**Comment by sclevine** [comment]
Wonderful, thanks! Sorry for the delay.

**Pull request merged**